SynthCh10Gen/Source_NasaNavTxt.cpp

```diff
@@ -9,7 +9,9 @@
 #include "Source_CsvTxt.h"
 #include "Source_NasaNavTxt.h"
 
+#ifndef LINUX_BUILD
```

Comment:
https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170

Reply:
The link does show me what that constant is, but it still broke the code when using cmake. So how would you recommend to fix it?